apps/api/src/handlers/createEvent/validateEvent.ts

```diff
@@ -0,0 +1,288 @@
+import { InputType } from '@aws-sdk/client-medialive';
+import {
+  CreateEventRequest,
+  InputNetworkLocation,
+  SrtDecryptionAlgorithm,
+} from '@trackflix-live/types';
+import { JSONSchemaType } from 'ajv';
+
```

Comment:
We have to find a way to avoid duplication here, it's would be crazy to have to edit all the schemas when we just want to make a small change